seedfarmer/cli_groups/_bootstrap_group.py

```diff
@@ -57,14 +57,40 @@ def bootstrap() -> None:
     help="Optionally also bootstrap the account as a Target account",
     required=False,
 )
+@click.option(
+    "--synth/--no-synth",
+    type=bool,
+    default=False,
+    help="Synthesize a CFN template only...do not deploy",
+    required=False,
+)
 @click.option("--debug/--no-debug", default=False, help="Enable detail logging", show_default=True)
 def bootstrap_toolchain(
-    project: str, trusted_principal: List[str], permission_boundary: Optional[str], as_target: bool, debug: bool
+    project: str,
+    trusted_principal: List[str],
+    permission_boundary: Optional[str],
+    as_target: bool,
+    synth: bool,
+    debug: bool,
 ) -> None:
     if debug:
         enable_debug(format=DEBUG_LOGGING_FORMAT)
     _logger.debug("Bootstrapping a Toolchain account for Project %s", project)
-    pass
+    bootstrap_toolchain_account(
+        project_name="exampleproj",
+        principalARN=trusted_principal,
+        permissionsBoundaryARN=permission_boundary,
+        synthesize=synth,
+    )
+    # if as_target:
```

Comment:
commented? is this not ready?

Reply:
No...need to get the account ID from the session, and as we discussed this morning we need to figure out exactly how we want to fetch that outside of the session manager